test/extensions/filters/http/cache/simple_http_cache/simple_http_cache_test.cc

```diff
@@ -37,7 +37,7 @@ class SimpleHttpCacheTest : public testing::Test {
   }
 
   // Updates the cache entry's header
-  void updateHeaders(LookupContextPtr lookup,
+  void updateHeaders(LookupContextPtr& lookup,
```

Comment:
Can this just be a `LookupContext&` instead of a reference to a unique_ptr? We dereference it on hte next line, anyway, so perhaps we just do that at the call sites for this `updateHeaders`.